File: javascript/numeros_pares.js
const n = parseInt(prompt("Quantos numeros voce vai digitar?"));

let vetor = [], qtdPares = 0;

for(i = 0; i < n; i++) {
    vetor[i] = parseInt(prompt("Digite um numero:"));

    if (vetor[i] % 2 == 0) {
        qtdPares++;
    }
}

console.log("NUMEROS PARES:");


for(i = 0; i < n; i++) {
    if (vetor[i] % 2 == 0) {
        console.log(vetor[i]);
    }
}

console.log("QUANTIDADE DE PARES = %d", qtdPares);

File: python/numeros_pares.py
n = int(input("Quantos numeros voce vai digitar? "))

vetor = []

for i in range(0, n):
    vetor.append(int(input("Digite um numero: ")))

print()

qtd_pares = 0
print("NUMEROS PARES:")
for i in range(0, n):
    if vetor[i] % 2 == 0:
        qtd_pares += 1
        print(vetor[i], end=" ")

print()
print()
print("QUANTIDADE DE PARES =", qtd_pares)
